core/core_string_names.h

```diff
@@ -44,45 +44,45 @@ class CoreStringNames {
 
 	_FORCE_INLINE_ static CoreStringNames *get_singleton() { return singleton; }
 
-	const StringName free_ = StaticCString::create("free"); // free would conflict with C++ keyword.
-	const StringName changed = StaticCString::create("changed");
-	const StringName script = StaticCString::create("script");
-	const StringName script_changed = StaticCString::create("script_changed");
-	const StringName _iter_init = StaticCString::create("_iter_init");
-	const StringName _iter_next = StaticCString::create("_iter_next");
-	const StringName _iter_get = StaticCString::create("_iter_get");
-	const StringName get_rid = StaticCString::create("get_rid");
-	const StringName _to_string = StaticCString::create("_to_string");
-	const StringName _custom_features = StaticCString::create("_custom_features");
+	const StringName free_ = StringName("free"); // free would conflict with C++ keyword.
```

Comment:
We can probably get rid of the casting redundancies while we're at it; I'm pretty sure none of these that specify only a string need to actually be wrapped with `StringName`.
```suggestion
	const StringName free_ = "free"; // free would conflict with C++ keyword.
```